<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<link rel="stylesheet" href="style.css">
<link rel="stylesheet" href="bootstrap/css/bootstrap.min.css">
<link rel="stylesheet" href="v2.1.7/fancybox-2.1.7/source/jquery.fancybox.css">
<title>GEMbench: Benchmark of Omics Data and Integration Methods</title>
<!-- Load d3.js -->
<script src="d3/d3.min.js"></script>
<script src="d3-tip.js"></script>
<script src="jquery-3.4.1.min.js"></script>
<link rel="shortcut icon" type="image/png" href="art/gembench-boxed.png"/>
</head>
<body>
  <header id="header" class="fixed-top">
    <div class="container-fluid d-flex">

      <div class="logo mr-auto">
        <a href="index.html"><img src="art/gembench.png" alt="" class="img-fluid"></a>
      </div>

      <nav class="nav-menu d-none d-lg-block">
        <ul>
          <li class="active"><a href="/#header">Home</a></li>
          <li><a href="/explore.html">Explore</a></li>
          <li><a href="/imprint.html">Imprint</a></li>
          <li><a href="/imprint.html#privacy">Privacy</a></li>
        </ul>
      </nav><!-- .nav-menu -->

    </div>
  </header>
  
  
  
  <section id="hero" class="d-flex align-items-center">

    <div class="container">
      <div class="row">
        <div class="col-lg-6 pt-5 pt-lg-0 order-2 order-lg-1">
          <h2>GEMbench: A comprehensive benchmark of omics data and integration methods for assessment of metabolic patterns in cancers</h2>
          <p>The cornerstone of metabolic research in oncology was laid by Otto Warburg in the 1920s when he discovered the main metabolic hallmark of cancer, aerobic glycolysis. Aerobic glycolysis (Warburg effect) refers to a common feature of cancer cell metabolism in which cancer cells take up glucose at high rates accompanied by lactate secretion in the presence of oxygen. Since then, a large quantity of high-throughput data has unravelled other metabolic hallmarks of cancer which contribute to better cancer stratification and personalized treatment. Aided by metabolic modelling approaches, in particular, genome-scale metabolic modelling (GEM), studies in cancer cell metabolism have expanded our understanding of the underlying mechanisms of cancer development, as well as identification of diagnostic biomarkers, potential drug targets, and prognosis biomarker. Using omics-data integration approaches, GEMs can be tailored to a specific context. These context-specific GEMs facilitate the unveiling of the genotype-phenotype relationship by calculating the flux states and extracting physiologically relevant information from high-quality data.</p>
          <p>Since 2002 to now, more than fifty methods have been developed to investigate how gene expression integration into GEMs could affect model content and predictive accuracy. These methods differ in assumptions and mathematical formulation, use different gene expression datasets (e.g. microarray data or RNA-sequencing data) and various GEMs of organisms from bacteria to human. These heterogeneous datasets and diverse algorithms are difficult to compare and to benchmark.</p>
          <p>Here, we performed a comprehensive benchmark to assess the influence of omics-data (regarding the data sources, types and platforms) and integration methods on the metabolic patterns in cancers. We collected different types of omics data (transcriptome and proteome) from different sources (cell-lines and patients), as well as different platforms (RNA-Seq and microarray) and integrated them into Recon3D (the generic GEM of Humans) by four integration algorithms (GIMME, iMAT, INIT, and FASTCORE).</p>
        </div>
        <div class="col-lg-6 order-1 order-lg-2 hero-img text-center">
          <img src="art/gembench-boxed.png" class="img-fluid mb-5" alt="">
          <p>Go ahead and interactively explore the results reported in  .....</p>
          <a href="/explore.html" class="btn">Get Started</a>
        </div>
      </div>
    </div>

  </section>
  
  
  
  <section id="details" class="d-flex align-items-center">
    <div class="container">
      <div class="section-title">
        <h2>Integration Methods</h2>
      </div>
      <p>
      From 2002 to now, more than fifty methods have been developed to investigate how gene expression integration into GEM could affect model content and predictive accuracy.
      The first study, which tried to overcome this issue, was published in 2002 by <a href="https://www.ncbi.nlm.nih.gov/pubmed/12006566">Covert and Palsson</a>.
      In 2004, <a href="https://www.ncbi.nlm.nih.gov/pubmed/15491858">Akesson et al.</a> used gene expression data as an additional constraint on the metabolic fluxes in yeast. Afterwards, different algorithms were devised for tackling this challenge <a href="https://www.ncbi.nlm.nih.gov/pubmed/18483554">GIMME</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/19714220">E-Flux</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/20823844">MBA</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/19346491">Moxley</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/21172910">MADE</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/23013597">RELATCH</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/22615553">INIT</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/23234303">mCADRE</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/24646661">tINIT</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/26942765">CORDA</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/27327084">E-Flux2</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/27327084">SPOT</a> and <a href="https://www.ncbi.nlm.nih.gov/pubmed/29222750">FASTCORE</a>. These algorithms can be classified in many different ways on the basis of the underlying methodology.
</p>
      <p>
 In 2014, <a href="https://www.ncbi.nlm.nih.gov/pubmed/24762745">Machado and Herrgård</a> classified methods by three main criteria: 1. how do they apply the gene expression levels (discrete vs continuous, absolute vs relative), 2. Model functionality regarding flux prediction or 3. Model building. 
</p>
      <p>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/25379144">Kim and Lun</a> considered four grouping criteria, regarding gene expression and model functionality: 1. Algorithms which require multiple gene expression datasets as input, 2. Requirement for gene expression threshold to define a gene's high and low expression, 3. Requirement of an appropriate objective function for a given GEM, and 4. Validation of predicted fluxes directly against measured intracellular fluxes.
</p>
      <p>
In 2015, <a href="https://www.ncbi.nlm.nih.gov/pubmed/26158726">Estévez and Nikoloski</a> stratified the methods in three distinct groups:
 <ul>
   <li><strong>GIMME- and GIM<sup>3</sup>E-like methods.</strong> This group refers to the two-steps approaches with a metabolic functionality (<em>objective function</em>) being optimized through <abbr title="Flux Balance Analysis">FBA</abbr> in a first step. Then, the obtained optimal value is employed to minimize the discrepancies between fluxes and data.</li>
   <li><strong>iMAT-and INIT-like methods.</strong>  This group of algorithms determines the binary reaction status (i.e., active or inactive) which are in good agreement with the associate data state.</li>
   <li><strong>MBA-, mCADRE and FASTCORE-like methods.</strong>  This group defines a core set of active reactions and then finds the minimum essential set of reactions to satisfy the model consistency condition.</li>
   </ul>
</p>
      <p>

In this study we focus on the following integration methods:

<ul>
  <li><strong>FASTCORE</strong> takes a core set of active reactions in the context of interest as an input</li>
  <li><strong>GIMME</strong> minimizes the utilization of inactive reactions</li>
  <li><strong>INIT</strong> uses the relaxation strategy to prevent the removal of reactions that have a small net accumulation rate</li>
  <li><strong>iMAT</strong> assumes gene expression into two groups: highly and lowly expressed and then maximizes the consistency with this classification</li>
</ul>

</p>


    </div>
  </section>
  <section id="details" class="d-flex align-items-center">
    <div class="container">
      <div class="section-title">
        <h2>Data</h2>
      </div>
      
      
  <div id="sunburst" class="m-5">
  <div id="sunburst-bc"></div>
    <div id="sunburst-explanation" style="visibility: hidden;">
      <span id="percentage"></span><br/>
      samples in this study
    </div>
  </div>
      
      <p>
        The data for our study was obtained from the following sources:
      </p>
      <div class="container">
      <div class="row">
        <div class="col-sm">
      <h4>EMTAB-37</h4>
        <ul>
          <li><strong>Source:</strong> Cell Line</li>
          <li><strong>Data type:</strong> Microarray data</li>
          <li><strong>Obtained:</strong> March 2019</li>
          <li><strong>From:</strong> <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-37/samples/">ebi.ac.uk/arrayexpress</a></li>
          <li><strong>Number of Samples:</strong> 317</li>
        </ul>
      <p>
        <a href="https://pubmed.ncbi.nlm.nih.gov/30357387">ArrayExpress</a> is an archive of functional genomics data that stores high-throughput functional genomics data and provides it for reuse to the research community.
        The transcriptomics profiles of various cancer cell lines were downloaded from <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-37/samples/">ebi.ac.uk/arrayexpress</a>.
        The gene expression profiles of 317 different cancer cell lines, categorized into 57 different pathological states and 28 individual tissues for study.
        In total 22&#x202f;012 unique genes exist in this dataset.
      </p>
      
    </div>
    <div class="col-sm">
      
      <h4>HPA</h4>
        <ul>
          <li><strong>Source:</strong> Cell Line</li>
          <li><strong>Data type:</strong> RNA Seq data</li>
          <li><strong>Obtained:</strong> March 2019</li>
          <li><strong>From:</strong> <a href="https://www.proteinatlas.org/about/download"></a></li>
          <li><strong>Number of Samples:</strong> 32</li>
        </ul>
      <p>
        The <a href="https://pubmed.ncbi.nlm.nih.gov/25613900">Human Protein Atlas (HPA)</a> is a freely available database including Tissue Atlas, Cell Atlas and Pathology Atlas.
        The Cell Atlas contains mRNA expression profiles of 64 cell lines which were characterized using deep RNA-sequencing.
        After filtering the non-metastatic human cancer cell lines, 32 cell lines were selected and the data files were downloaded.
      </p>
      
    </div>
    <div class="col-sm">
      
      <h4>ProteomeNCI60 </h4>
        <ul>
          <li><strong>Source:</strong> Cell Line</li>
          <li><strong>Data type:</strong> Mass Spec Proteomics data</li>
          <li><strong>Obtained:</strong> March 2019</li>
          <li><strong>From:</strong> <a href="https://www.proteomicsdb.org/#projects/35/256"></a></li>
          <li><strong>Number of Samples:</strong> 59</li>
        </ul>
      <p>
        <a href="https://www.proteomicsdb.org/">ProteomicsDB</a> is an effort of the Technische Universität München (TUM) that dedicated to expedite the identification of various proteomes and their use across the scientific community.
        We used the results of the Global Proteome Analysis study.
        <a href="https://pubmed.ncbi.nlm.nih.gov/23933261">Gholami et al.</a> employed an MS-based proteomics analysis and provided quantitative proteome profiles of all 59 cell lines of the NCI-60 panel.
        The dataset contains 10&#x202f;350 identified proteins. 
      </p>
      
    </div>
  </div>
  
      <div class="row">
        <div class="col-sm">
      
      <h4>GSE2109</h4>
        <ul>
          <li><strong>Source:</strong> Patient Data</li>
          <li><strong>Data type:</strong> Microarray data</li>
          <li><strong>Obtained:</strong> March 2019</li>
          <li><strong>From:</strong> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2109">ncbi.nlm.nih.gov</a></li>
          <li><strong>Number of Samples:</strong> 315</li>
        </ul>
      <p>
        The <a href="https://pubmed.ncbi.nlm.nih.gov/11752295">Gene Expression Omnibus (GEO)</a> is a public functional genomics data repository that stores microarray- and sequence-based data.
        The results of DNA microarray analysis of The Expression Project for Oncology (expO) from The International Genomics Consortium were downloaded from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2109">ncbi.nlm.nih.gov</a>.
        From 2&#x202f;158 patient samples, the metastatic and ambiguous samples were removed and 1,895 samples in 315 different cancer types were selected.
        In total, 24&#x202f;442 unique genes exist in this dataset. 
      </p>
      
    </div>
    <div class="col-sm">
      
      <h4>TCGA</h4>
        <ul>
          <li><strong>Source:</strong> Patient Data</li>
          <li><strong>Data type:</strong> RNA Seq data</li>
          <li><strong>Obtained:</strong> March 2019</li>
          <li><strong>From:</strong> <a href="https://portal.gdc.cancer.gov/legacy-archive/search/f">portal.gdc.cancer.gov</a></li>
          <li><strong>Number of Samples:</strong> 202</li>
        </ul>
      <p>
        The <a href="https://www.cancer.gov/tcga">Cancer Genome Atlas (TCGA)</a> is a joint effort between the National Cancer Institute and the National Human Genome Research Institute that generated high-throughput data.
        The RNA-seq data was obtained using the GDC Data Portal.
        The portal has been searched for all RNA-seq cases and HTSeq-FPKM workflow type.
        In total, 11&#x202f;571 files for 10&#x202f;672 cases were found.
        The metadata file was downloaded and only files of primary tumour or primary blood derived cancer bone marrow or peripheral were selected.
        Finally, 10&#x202f;322 files were downloaded through their API.
        We combined the same disease types by averaging the expression values and obtained 202 samples of different cancer types for the current study.
      </p>
      
    </div>
    <div class="col-sm">
      
      <h4>ProteomePatients</h4>
        <ul>
          <li><strong>Source:</strong> Patient Data</li>
          <li><strong>Data type:</strong> Mass Spec Proteomics data</li>
          <li><strong>Obtained:</strong> March 2019</li>
          <li><strong>From:</strong> <a href=""></a></li>
          <li><strong>Number of Samples:</strong> 10</li>
        </ul>
      <p>
        The <a href="http://www.proteomexchange.org">ProteomeXchange (PX) Consortium</a> provided a standard portal for submission and dissemination of mass spectrometry proteomics data.
        We extracted 10 data sets of four different cancer types containing 11&#x202f;961 identified proteins.
      </p>
    </div>
    
    </div>
    </div>
  </div>
  </section>
    
  <footer id="footer" class="text-center">
    <div class="container py-4">
      <p class="">The <a href="/imprint#imprint">GEMbench</a> is a project of the <a href="https://www.sbi.uni-rostock.de/">Department of Systems Biology and Bioinformatics</a> at the <a href="http://www.uni-rostock.de/">University of Rostock</a></p>
    </div>
  </footer>
<script src="index.js"></script>
</body>
</html>
