<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<link rel="stylesheet" href="style.css">
<link rel="stylesheet" href="bootstrap/css/bootstrap.min.css">
<link rel="stylesheet" href="v2.1.7/fancybox-2.1.7/source/jquery.fancybox.css">
</head>
<body>
  <header id="header" class="fixed-top">
    <div class="container-fluid d-flex">

      <div class="logo mr-auto">
        <a href="index.html"><img src="art/gembench.png" alt="" class="img-fluid"></a>
      </div>

      <nav class="nav-menu d-none d-lg-block">
        <ul>
          <li class="active"><a href="/#header">Home</a></li>
          <li><a href="/explore.html">Explore</a></li>
          <li><a href="/imprint.html">Imprint</a></li>
          <li><a href="/imprint.html#privacy">Privacy</a></li>
        </ul>
      </nav><!-- .nav-menu -->

    </div>
  </header>
  
  
  
  <section id="hero" class="d-flex align-items-center">

    <div class="container">
      <div class="row">
        <div class="col-lg-6 pt-5 pt-lg-0 order-2 order-lg-1">
          <h2>GEMbench:
<!--
          </h1>
-->
<!--
          <h2>
-->
            A comprehensive benchmark of omics data and integration methods for assessment of metabolic patterns in cancers</h2>
          <p>The ultimate goal of systems medicine is to understand the underlying mechanism of disease development, diagnostic biomarkers, potential drug targets and prognosis biomarkers. The generic genome-scale metabolic models (GEMs) is one of the most prevalent networks that in integration with omics-data could generate the context-based/tissue-specific GEM. Generating the context-based GEM has allowed a more in-depth study of metabolism and in particular, these facilitate the unveiling of the genotype-phenotype relationship and give rise to the extraction of physiologically relevant information from the high-quality data.</p>
          <p>Since 2002 to now, more than fifty methods have been developed to investigate how gene expression integration into GEM could affect model content and predictive accuracy. These methods differ in assumptions and mathematical formulation, used different gene expression datasets (in form of microarray data or RNA-sequencing data) and various GEMs of organisms from bacteria to human as well as tested isolated case studies. These heterogeneous datasets and diverse algorithms are difficult for comparison and benchmarking.</p>
          <p>Here, we performed a comprehensive benchmark to assess the influence of omics-data (regarding to the data sources, types and platforms) and integration methods on the model content and predictive accuracy. We collected different types of omics data (Transcriptome and proteome) from different sources (cell-lines and patients), as well as different platforms (RNA-Seq and Microarray) and integrated them into Recon3D (the generic GEM of Human) by four integration algorithms (GIMME, iMAT, INIT, and FASTCORE).</p>
        </div>
        <div class="col-lg-6 order-1 order-lg-2 hero-img text-center">
          <img src="art/gembench-boxed.png" class="img-fluid animated mb-5" alt="">
          <p>Go ahead and interactively explore the results reported in  .....</p>
          <a href="/explore.html" class="btn">Get Started</a>
        </div>
      </div>
    </div>

  </section>
  
  
  <section id="details" class="d-flex align-items-center">
    <div class="container">
      <div class="section-title">
        <h2>Background on Integration Methods</h2>
      </div>
      <p>
      From 2002 to now, more than fifty methods have been developed to investigate how gene expression integration into GEM could affect model content and predictive accuracy.
      The first study, which tried to overcome this issue, was published in 2002 by <a href="https://www.ncbi.nlm.nih.gov/pubmed/12006566">Covert and Palsson</a>.
      In 2004, <a href="https://www.ncbi.nlm.nih.gov/pubmed/15491858">Akesson et al.</a> used gene expression data as an additional constraint on the metabolic fluxes in yeast. Afterwards, different algorithms were devised for tackling this challenge <a href="https://www.ncbi.nlm.nih.gov/pubmed/18483554">GIMME</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/19714220">E-Flux</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/20823844">MBA</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/19346491">Moxley</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/21172910">MADE</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/23013597">RELATCH</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/22615553">INIT</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/23234303">mCADRE</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/24646661">tINIT</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/26942765">CORDA</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/27327084">E-Flux2</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/27327084">SPOT</a> and <a href="https://www.ncbi.nlm.nih.gov/pubmed/29222750">FASTCORE</a>. These algorithms can be classified in many different ways on the basis of the underlying methodology.
</p>
      <p>
 In 2014, <a href="https://www.ncbi.nlm.nih.gov/pubmed/24762745">Machado and Herrgård</a> classified methods by three main criteria: 1. Regarding how they apply the gene expression levels (discrete vs continuous, absolute vs relative), 2. Model functionality regarding flux prediction or 3. Model building. 
</p>
      <p>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/25379144">Kim and Lun</a> considered four grouping criteria, regarding gene expression and model functionality: 1. Algorithms which require multiple gene expression datasets as input, 2. Requirement for gene expression threshold to define a gene's high and low expression, 3. Requirement of an appropriate objective function for a given GEM, and 4. Validation of predicted fluxes directly against measured intracellular fluxes.
</p>
      <p>
In 2015, <a href="https://www.ncbi.nlm.nih.gov/pubmed/26158726">Estévez and Nikoloski</a> stratified the methods in three distinct groups. 1. GIMME- and GIM<sup>3</sup>E-like methods. This group refers to the two-steps approaches with a metabolic functionality (<em>objective function</em>) being optimized through <abbr title="Flux Balance Analysis">FBA</abbr> in a first step. Then, the obtained optimal value is employed to minimize the discrepancies between fluxes and data. 2. iMAT-and INIT-like methods. This group of algorithms determines the binary reaction status (i.e., active or inactive) which are in good agreement with the associate data state. 3. MBA-, mCADRE and FASTCORE-like methods. This group defines a core set of active reactions and then finds the minimum essential set of reactions to satisfy the model consistency condition.
</p>
      <p>

In this study we focus on the following integration methods:

<ul>
  <li><strong>FASTCORE</strong> takes a core set of active reactions in the context of interest as an input</li>
  <li><strong>GIMME</strong> minimizes the utilization of inactive reactions</li>
  <li><strong>INIT</strong> uses the relaxation strategy to prevent the removal of reactions that have a small net accumulation rate</li>
  <li><strong>iMAT</strong> assumes gene expression into two groups: highly and lowly expressed and then maximizes the consistency with this classification</li>
</ul>

</p>


    </div>
  </section>
    
  <footer id="footer" class="text-center">
    <div class="container py-4">
      <p class="">The <a href="/imprint#imprint">GEMbench</a> is a project of the <a href="https://www.sbi.uni-rostock.de/">Department of Systems Biology and Bioinformatics</a> at the <a href="http://www.uni-rostock.de/">University of Rostock</a></p>
    </div>
  </footer>
</body>
</html>
